modules/sync/syncs/httpsynchronization.cpp

```diff
@@ -222,6 +331,28 @@ bool HttpSynchronization::trySyncFromUrl(std::string listUrl) {
     }
     startedAllDownloads = true;
 
+    //TODO: Set number of retries from e.g., cfg or other setting?
+    int downloadRetries = 10;
```

Comment:
```suggestion
    constexpr int MaxDownloadRetries = 10;
```

for example.  Marking it as `const` or `constexpr` signals that it won't be changed